hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/OzoneContainer.java

```diff
@@ -263,15 +269,24 @@ public OzoneContainer(HddsDatanodeService hddsDatanodeService,
             blockDeletingServiceWorkerSize, config,
             datanodeDetails.threadNamePrefix(),
             context.getParent().getReconfigurationHandler());
+    Clock clock = Clock.system(ZoneId.systemDefault());
 
     Duration diskBalancerSvcInterval = conf.getObject(
         DiskBalancerConfiguration.class).getDiskBalancerInterval();
     Duration diskBalancerSvcTimeout = conf.getObject(
         DiskBalancerConfiguration.class).getDiskBalancerTimeout();
+
+    replicationSupervisor = ReplicationSupervisor.newBuilder()
```

Comment:
No need to create a new ReplicationSupervisor.  This newly created ReplicationSupervisor will not be used by replication. 

get the ReplicationSupervisor from hddsDatanodeService.